scm-ui/ui-webapp/src/repos/containers/RepositoryRoot.tsx

```diff
@@ -292,6 +293,10 @@ const RepositoryRoot = () => {
                 exact={true}
                 render={() => <TagsOverview repository={repository} baseUrl={`${url}/tag`} />}
               />
+              <Route
+                path={`${url}/compare/:sourceType/:sourceName`}
+                render={() => <CompareRoot repository={repository} baseUrl={`${url}/compare`} />}
```

Comment:
Please use children. I think we should change the other routes in this file too.